testdata/not-the-leader/inputs.json

```diff
@@ -0,0 +1,105 @@
+{
+   "Now": "2020-11-02T15:00:00Z",
+   "FirstStateTime": "2020-11-02T13:12:34Z",
+   "Config": {
+      "CleanupDeadServers": true,
+      "LastContactThreshold": 200000000,
+      "MaxTrailingLogs": 200,
+      "MinQuorum": 3,
+      "ServerStabilizationTime": 10000000000
+   },
+   "State": {
+      "Healthy": false
+   },
+   "RaftConfig": {
+      "Servers": [
+         {
+            "Suffrage": 0,
+            "ID": "7875975d-d54b-49c1-a400-9fefcc706c67",
+            "Address": "198.18.0.1:8300"
+         },
+         {
+            "Suffrage": 0,
+            "ID": "ecfc5237-63c3-4b09-94b9-d5682d9ae5b1",
+            "Address": "198.18.0.2:8300"
+         },
+         {
+            "Suffrage": 0,
+            "ID": "e72eb8da-604d-47cd-bd7f-69ec120ea2b7",
+            "Address": "198.18.0.3:8300"
+         }
+      ]
+   },
+   "KnownServers": {
+      "7875975d-d54b-49c1-a400-9fefcc706c67": {
+         "ID": "7875975d-d54b-49c1-a400-9fefcc706c67",
+         "Name": "node1",
+         "Address": "198.18.0.1:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      },
+      "ecfc5237-63c3-4b09-94b9-d5682d9ae5b1": {
+         "ID": "ecfc5237-63c3-4b09-94b9-d5682d9ae5b1",
+         "Name": "node2",
+         "Address": "198.18.0.2:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      },
+      "e72eb8da-604d-47cd-bd7f-69ec120ea2b7": {
+         "ID": "e72eb8da-604d-47cd-bd7f-69ec120ea2b7",
+         "Name": "node3",
+         "Address": "198.18.0.3:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      }
+   },
+   "AliveServers": {
+      "7875975d-d54b-49c1-a400-9fefcc706c67": {
+         "ID": "7875975d-d54b-49c1-a400-9fefcc706c67",
+         "Name": "node1",
+         "Address": "198.18.0.1:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      },
+      "ecfc5237-63c3-4b09-94b9-d5682d9ae5b1": {
+         "ID": "ecfc5237-63c3-4b09-94b9-d5682d9ae5b1",
+         "Name": "node2",
+         "Address": "198.18.0.2:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      },
+      "e72eb8da-604d-47cd-bd7f-69ec120ea2b7": {
+         "ID": "e72eb8da-604d-47cd-bd7f-69ec120ea2b7",
+         "Name": "node3",
+         "Address": "198.18.0.3:8300",
+         "NodeStatus": "alive",
+         "Version": "1.9.0",
+         "RaftVersion": 3
+      }
+   },
+   "LatestIndex": 500,
```

Comment:
There are two things to note with this test. First is that the node running autopilot is 524 logs behind the leader. We can see that with the top level `LatestIndex` being 500 but also within the third server in the `FetchedStats` map. The second important thing is that `IsLeader` is set to `false`. These combined will force autopilot to use the last term/index from the fetched stats of the leader instead of the top level versions.

Eventually this will cause the server to mark itself as unhealthy as it is too far behind.